src/app/qgsattributetabledialog.h

```diff
@@ -69,44 +69,77 @@ class APP_EXPORT QgsAttributeTableDialog : public QDialog, private Ui::QgsAttrib
      */
     void editingToggled();
 
+    /**
+     * @brief Called when the loading of features has been completed
+     */
+    void loadFinished();
+
+    /**
+     * @brief Called while features are being loaded, cancel to abort
+     * @param i
```

Comment:
Can you include a comment as to what *i* is? I think it's the "percentDone" right or is it the count?

Or you could save yourself a comment by calling the var "percentDone" instead of "i".